src/scss/tooltip.scss

```diff
@@ -194,10 +194,4 @@
 	padding: 0;
 	margin: 5px 5px 15px 15px;
 	border: 0;
-
-	&:focus {
```

Comment:
Would removing a custom focus style count as a breaking change?

Reply:
I think not because you'll just get the browser default or a globally set alternative instead.